test/parallel/test-http-mutable-headers.js

```diff
@@ -43,8 +43,9 @@ const s = http.createServer(common.mustCall((req, res) => {
   switch (test) {
     case 'headers':
       // Check that header-related functions work before setting any headers
-      // eslint-disable-next-line no-restricted-properties
-      assert.deepEqual(res.getHeaders(), {});
+      const headers = res.getHeaders();
+      const exoticObj = Object.create(null);
```

Comment:
I agree with @thefourtheye. Probably better to call it something like `simpleObj` or `bareObj` or something similar without getting too wordy...